src/pages/patientView/mutation/PatientViewMutationTable.tsx

```diff
@@ -79,7 +79,7 @@ export enum MutationTableColumnType {
 
 type MutationTableColumn = Column<Mutation[]>&{order:number};
 
-class MutationTableComponent extends MSKTable<Mutation[]> {
+class MutationTableComponent extends LazyMobXTable<Mutation[]> {
```

Comment:
Changes in this component will definitely conflict with the PR #320, depending on which PR we are gonna merge first, the other one needs to be rebased and modified accordingly.